library/src/main/java/io/codearte/props2yaml/Props2YAML.java

```diff
@@ -1,15 +1,45 @@
 package io.codearte.props2yaml;
 
+import java.io.File;
+import java.io.FileInputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.StringReader;
+import java.nio.file.Path;
 import java.util.Properties;
 
 public class Props2YAML {
 
     private final Properties properties = new Properties();
 
-    public Props2YAML(String source) {
-        fillProperties(source);
+    Props2YAML(String source) {
+        try {
+            properties.load(new StringReader(source));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    Props2YAML(File file) {
```

Comment:
There is a duplication between constructors. The types are different, but maybe it could be refactored somehow?